src/net.cpp

```diff
@@ -36,6 +36,7 @@
 #include <miniupnpc/miniwget.h>
 #include <miniupnpc/upnpcommands.h>
 #include <miniupnpc/upnperrors.h>
+static_assert(MINIUPNPC_API_VERSION >= 10, "miniUPnPc API version >= 10 assumed");
```

Comment:
There should be a code comment explaining this version requirement so we can remember the reasoning behind it in the future. You could copy and paste the text from the release-notes here.

Reply:
Done.